code/game/area/LV759_Hybrisa_Prospera.dm

```diff
@@ -0,0 +1,939 @@
+//lv759 AREAS--------------------------------------//
+
+/area/lv759
+	name = "LV-759 Hybrisa Prospera"
+	icon = 'icons/turf/hybrisareas.dmi'
+	icon_state = "hybrisa"
+	can_build_special = TRUE
+	powernet_name = "ground"
+	minimap_color = MINIMAP_AREA_COLONY_RESANDCOM
+
+//parent types
+
+/area/lv759/indoors
+	name = "Hybrisa - Indoors"
+	icon_state = "cliff_blocked"//because this is a PARENT TYPE and you should not be using it and should also be changing the icon!!!
+	ceiling = CEILING_METAL
+	soundscape_playlist = SCAPE_PL_LV759_INDOORS
+	ambience_exterior = AMBIENCE_HYBRISA_INTERIOR
+/area/lv759/outdoors
+	name = "Hybrisa - Outdoors"
+	icon_state = "cliff_blocked"//because this is a PARENT TYPE and you should not be using it and should also be changing the icon!!!
+	ceiling = CEILING_NONE
+	soundscape_playlist = SCAPE_PL_LV759_OUTDOORS
+	ambience_exterior = AMBIENCE_CITY
+	soundscape_interval = 25
+/area/lv759/oob
+	name = "Out Of Bounds"
+	icon_state = "oob"
+	ceiling = CEILING_MAX
+	is_resin_allowed = FALSE
+	flags_area = AREA_NOTUNNEL
+	minimap_color = MINIMAP_AREA_OOB
+
+// Landing Zone 1
+/area/lv759/outdoors/landing_zone_1
+	name = "Nova Medica Hospital Complex - Emergency Response Landing Zone One"
+	icon_state = "medical_lz1"
+	is_resin_allowed =  FALSE
+	is_landing_zone = TRUE
+	minimap_color = MINIMAP_AREA_LZ
+	ceiling = CEILING_NONE
+/area/lv759/indoors/landing_zone_1/flight_control_room
+	name = "Nova Medica Hospital Complex - Emergency Response Landing Zone One - Flight Control Room"
+	icon_state = "hybrisa"
+	ceiling = CEILING_METAL
+/area/lv759/indoors/landing_zone_1/lz1_console
+	name = "Nova Medica Hospital Complex - Emergency Response Landing Zone One - Dropship Alamo Console"
+	icon_state = "hybrisa"
+	requires_power = FALSE
+	ceiling = CEILING_METAL
+
+// Landing Zone 2
+/area/lv759/outdoors/landing_zone_2
+	name = "KMCC Interstellar Freight Hub - Landing Zone Two"
+	icon_state = "mining_lz2"
+	is_resin_allowed =  FALSE
+	is_landing_zone = TRUE
+	minimap_color = MINIMAP_AREA_LZ
+	ceiling = CEILING_NONE
+/area/lv759/indoors/landing_zone_2/kmcc_hub_flight_control_room
+	name = "KMCC Interstellar Freight Hub - Flight Control Room"
+	icon_state = "hybrisa"
+	ceiling = CEILING_METAL
+/area/lv759/indoors/landing_zone_2/kmcc_hub_security
```

Comment:
You maybe should consider adding `is_landing_zone = TRUE` to this area. Unless you shrink the LZ2 area to not cut off this area as much.